installerconfig

```diff
@@ -75,7 +75,7 @@ env PAGER=cat freebsd-update -f /tmp/freebsd-update.conf install
 echo "==> Installing packages"
 env ASSUME_ALWAYS_YES=YES pkg update -f
 env ASSUME_ALWAYS_YES=YES pkg upgrade -q -y
-env ASSUME_ALWAYS_YES=YES pkg install -q -y bash curl node npm wget
+env ASSUME_ALWAYS_YES=YES pkg install -q -y bash curl node npm wget vim-console tmux
```

Comment:
I'm not sure we necessarily want to include these. Some users may have preferences for other things. The items already always included are necessary for Joyent images, which is why they are there.

The rest of this PR looks good though, so if you can fix this then we'll get some builds and test it out.